### DISCUSSION TITLE
Any idea how to use directional buttons to make dialog choices? (Not hotkeys)

### DISCUSSION THREAD

**Discussion by Kodykthnx** [dialogic-godot/dialogic#886] [Q&A] [answered]
I want to be able to cycle through the dialogue options with the arrow keys/d-pad/analog stick, and not with my mouse. From the looks of things, there appears to be no way to do this (though I could be wrong.) I'd greatly appreciate it if someone could tell me a simple solution to this. I'm new to gamedev in general, so my apologies if I'm being a nuisance lol
Thanks much!

edit: i did search both here and in the archived discord help channel before asking, couldn't find a solution that worked, just to clarify

**Comment by zaknafean** [comment, marked as answer]
It actually should work right out of the box if you haven't played with your settings. 
![image](https://user-images.githubusercontent.com/7741797/166230881-840486ec-4f44-4518-894d-6f27e715f2ff.png)

If that's checked, then it will just work, and you can use the keyboard/controller. BUT there doesn't appear to be any indicator of what's selected, so you can't tell that a choice has focus. That's a bit of a known issue we've been putting off, and while I don't have an ETA on a fix for it, you can side step it by doing the following:
1) Go to 'ChoiceButton.tscn' located at **res://addons/dialogic/Nodes/ChoiceButton.tscn**
2) Connect the 'focus_entered' and 'focus_exited' signals and modulate the button to a color of your choice.

You can do  2 by adding the following code to **ChocieButton.gd**
![Screenshot 2022-05-02 081711](https://user-images.githubusercontent.com/7741797/166232163-ff5b0fe9-ca63-4c6f-a4f4-5501f00e2503.png)

**Reply by Kodykthnx** [reply]
Ah, okay I see. It was working but I couldn't see what was selected. Thanks for the fix! Works great now.
